README.rst

```diff
@@ -1,6 +1,9 @@
 Toil is a scalable, efficient, cross-platform pipeline management system, written entirely in
 Python, and designed around the principles of functional programming. Full documentation can be found at `Read the Docs`_.
 
+Native system requirements: `python2.7-dev`_, make_, gcc_, `libffi-dev`_,
```

Comment:
And I would use generic names for these: "Python headers and static libraries", "Make", "GCC", and libffi library and headers. Then mention the steps that needs to be run to get these on Ubuntu and OS X.